GoogleForCreators/web-stories-wp
# Custom Fonts: Add error handling for duplicates

## Context



## Summary

Errors when trying to add the same font again, are not currently handled. Show a error message when font is trying to be re-added. 

## Relevant Technical Choices



## To-do



## User-facing changes


<img width="672" alt="Screenshot 2022-03-21 at 12 20 51" src="https://user-images.githubusercontent.com/237508/159261508-ca813622-6f33-43cf-bd03-200bdd8e5ec6.png">


## Testing Instructions






This PR can be tested by following these steps:
1. Enable custom font experiment. 
2. Login as admin and go to dashboard.
3. add font https://overpass-30e2.kxcdn.com/overpass-regular.ttf
4. See font being added
5. Add same font https://overpass-30e2.kxcdn.com/overpass-regular.ttf
6. See error message. 

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #10941
